portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -24,9 +28,26 @@
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
 
+  private ArrayList<Comment> comments = 
+    new ArrayList<Comment>(
+        Arrays.asList(new Comment("Comm1"), new Comment("Comm2"), new Comment("Comm3")));
+
```

Comment:
It's probably temporary, I just followed the [walkthrough](https://github.com/PredaMihaiDragos/Google-STEP/blob/master/walkthroughs/week-3-server/step-3-json-walkthrough.md)

> Modify DataServlet to contain an ArrayList<String> variable containing a few hard-coded messages. Keep it simple! An ArrayList containing three hard-coded values is good enough.

I made my own comment class because I will probably want later to add more variables to a comment (name, message, date etc)